dag/struct-root.go

```diff
@@ -12,7 +12,8 @@ type Root struct {
 func (r *Root) PrintDag(i interface{}) {
 	fmt.Println("Root:", r.Name, "\n")
 	for _, m := range r.Members {
-		m.GetValue(i)
+		m.GetValue(i, 0)
+		fmt.Println("")
```

Comment:
What is the purpose of this print?

Reply:
It print the created DAG based on the cue program passed as argument

Reply:
![Screenshot from 2022-05-22 21-50-28](https://user-images.githubusercontent.com/71719097/169713301-16a42fa9-e9e6-4d45-a9f5-0d953100b765.png)

Reply:
Awesome!